src/w3c/templates/headers.js

```diff
@@ -7,7 +7,20 @@ export default conf => {
   const html = hyperHTML;
   return html`<div class='head'>
   ${conf.logos.map(showLogo)}
-  <h1 class='title p-name' id='title'>${conf.title}</h1>
+  <script>
```

Comment:
Not quite right. You need to look at what the `html` tagged template string is doing 😉 

We definitely don’t want a `script` element here.